carthage/openssl-static-xcframework.json

```diff
@@ -1,4 +1,4 @@
 {
-    "1.1.11101": "https://github.com/cossacklabs/openssl-apple/releases/download/v1.1.11101/openssl-static-xcframework.zip",
-    "1.1.10803": "https://github.com/cossacklabs/openssl-apple/releases/download/v1.1.10803/openssl-static-xcframework.zip",
+    "1.1.11101": "https://github.com/cossacklabs/openssl-apple/releases/download/1.1.11101/openssl-static-xcframework.zip",
+    "1.1.10803": "https://github.com/cossacklabs/openssl-apple/releases/download/1.1.10803/openssl-static-xcframework.zip",
```

Comment:
`openssl-dynamic-xcframework.json` would probably need the same changes.

Are we even using that file? Do we intend to support it?

Reply:
Yeah, fixed it for dymanic xcf as well.
We are not using it right now but it is better to keep it as long as it doesn't require extra effort to support it. We may need it in the future because we can't predict if it is better than a static one for SPM and Carthage.